addons/interaction/CfgVehicles.hpp

```diff
@@ -253,40 +253,47 @@ class CfgVehicles {
                 modifierFunction = QUOTE([ARR_3(assignedTeam _target,'PATHTOF(UI\team\team_management_ca.paa)',_this select 3)] call FUNC(modifyTeamManagementAction));
                 showDisabled = 1;
 
+                class ACE_remoteTeamManagement {
+                    displayName = CSTRING(Squad);
+                    icon = QPATHTOF(UI\team\team_management_ca.paa);
+                    condition = QUOTE(GVAR(remoteTeamManagement));
```

Comment:
should this have `(_unit == leader _unit)`?

Reply:
It probably should.